pkgs/applications/misc/swaynotificationcenter/default.nix

```diff
@@ -30,15 +32,43 @@ stdenv.mkDerivation rec {
     hash = "sha256-79Kda2Mi2r38f0J12bRm9wbHiZCy9+ojPDxwlFG8EYw=";
   };
 
-  nativeBuildInputs = [ gobject-introspection meson ninja pkg-config scdoc vala wrapGAppsHook ];
+  nativeBuildInputs = [
+    bash-completion
+    # cmake # currently conflicts with meson
+    fish
+    glib
+    gobject-introspection
+    # installShellFiles # already installing automatically
+    meson
+    ninja
+    pkg-config
+    scdoc
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    dbus
+    dbus-glib
+    gdk-pixbuf
+    glib
+    gtk-layer-shell
+    gtk3
+    json-glib
+    libhandy
+    librsvg
+    # systemd # ends with broken permission
+  ];
 
-  buildInputs = [ dbus dbus-glib gdk-pixbuf glib gtk-layer-shell gtk3 json-glib libhandy librsvg ];
+  postInstall = ''
+    glib-compile-schemas "$out"/share/glib-2.0/schemas
+  '';
 
```

Comment:
Typically, we patch the `postinstall.py` (or the equivalent) that is usually part of meson/gtk projects.

```nix
postPatch = ''
  chmod +x build-aux/meson/postinstall.py
  # Requires python3 in nativeBuildInputs
  patchShebangs build-aux/meson/postinstall.py
'';
```

This pattern is used in many other packages.

Reply:
Great catch

Reply:
So, we have `build-aux/meson/postinstall.py` in the `0.6.3` tag, but the line `meson.add_install_script('build-aux/meson/postinstall.py')` it's only in `master`.

Reply:
@berbiche I've added notes for future version bumping.